Lib/pdb.py

```diff
@@ -1890,11 +1997,16 @@ def set_trace(*, header=None):
 # Post-Mortem interface
 
 def post_mortem(t=None):
-    """Enter post-mortem debugging of the given *traceback* object.
+    """Enter post-mortem debugging of the given *traceback*, or *exception*
+    object.
 
     If no traceback is given, it uses the one of the exception that is
     currently being handled (an exception must be being handled if the
     default is to be used).
+
+    If `t` is an Exception and is a chained exception (i.e it has a __context__,
+    or a __cause__), pdb will be able to list and move to other exceptions in
+    the chain using the `exceptions` command
```

Comment:
I think we just need a hint here, so tried to make it shorter (anyone interested in more details can check the docs).

```suggestion
    If `t` is an exception object, the `exceptions` command makes it possible to
    list and inspect its chained exceptions (if any).
```